submitit/auto/auto.py

```diff
@@ -171,7 +171,12 @@ def _internal_update_parameters(self, **kwargs: Any) -> None:
                 )
                 continue
         if invalid:
-            invalid.append(f"Known executors: {', '.join(executors.keys())}")
+            invalid.extend(
```

Comment:
this seems unrelated ? did you intend to push it ? looks good otherwise

Reply:
Not related indeed, but I thought about improving the error message since I kept getting "unknown cluster 'tasks'" which I found unhelpful. The issue was that I was using `tasks_per_job` instead of `tasks_per_node`